pom.xml

```diff
@@ -362,6 +330,50 @@
     </plugins>
   </build>
   <profiles>
+    <profile>
+      <id>spotless</id>
+      <activation>
+        <file>
+          <exists>contributing</exists>
```

Comment:
@csviri because of this. Basically, only run spotless if the `contrinbuting` dir is correctly set. So I can do for instance: 

```
cd operator-framework
mvn clean install -Pintegration-tests -Dtest=InformerRemoteClusterIT